rotkehlchen/api/rest.py

```diff
@@ -1967,6 +1967,47 @@ def edit_xpub(
 
         return api_response(process_result(_wrap_in_result(data, '')), status_code=HTTPStatus.OK)
 
+    def _add_evm_accounts(
+            self,
+            account_data: list[BlockchainAccountData],
+    ) -> dict[str, Any]:
+        try:
+            added_accounts = self.rotkehlchen.add_evm_accounts(account_data=account_data)
+        except EthSyncError as e:
+            return {'result': None, 'message': str(e), 'status_code': HTTPStatus.CONFLICT}
+        except TagConstraintError as e:
+            return {'result': None, 'message': str(e), 'status_code': HTTPStatus.CONFLICT}
```

Comment:
This two can be combined in one single except clause

Reply:
good point. Thanks.